public/index.html

```diff
@@ -15,6 +15,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
If you are using libraries, try to add all of them in the package.json file, so its version can be handled more efficiently for future project maintenance and to give it visibility about what is used in the project. The package.json is the truth source of the installed libraries.

Reply:
I added the library into package.json and removed the import in index.html